tests/extendrtests/src/rust/src/attributes.rs

```diff
@@ -0,0 +1,47 @@
+use extendr_api::prelude::*;
+
+/// Adds an attribute to a vector of doubles
+/// 
+/// @param x Vector of doubles 
+/// 
+/// @export
```

Comment:
I highly recommend not exporting functions. #[extendr] macro makes it available for testing on it is own.
Exported things has to be documented, as per `r-cmd-check`.
This means I've had to endure a cycle of:
```sh
cargo extendr document
cargo extendr devtools-test
cargo extendr devtools-test -a
cargo extendr r-cmd-check
```

To ensure that this was documented appropriately, to pass CI.